### ISSUE TITLE
[Accessibility][PDF Viewer] "Enable accessibility for screen readers" control cannot be unchecked

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1742]
**Steps to Reproduce**
1. Enable NVDA screen reader.
2. Open page in Google Chrome.
3. Navigate to menu controls for In-Line PDF Viewer.
4. Locate and activate "Enable accessibility for screen readers" checkbox (visually hidden).
5. Press Space bar to check the checkbox. 
6. Press Space bar again to attempt to deactivate accessibility mode.

**Current Behavior**
The "Enable accessibility for screen readers" checkbox can be checked with the Space bar, but cannot be unchecked with the keyboard.

**Expected behavior/code**
The "Enable accessibility for screen readers" checkbox can be both checked and unchecked using keyboard alone.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Keyboard users, screen reader users

**WCAG SC**
2.1.1 Keyboard

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
